Upstream-Status: Backport

Signed-off-by: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
---
From 9ca5a503b8f28258126226df203bef7fd316dfad Mon Sep 17 00:00:00 2001
From: "raphael.kubo.da.costa" <raphael.kubo.da.costa@intel.com>
Date: Thu, 11 Aug 2016 16:35:09 -0700
Subject: [PATCH] gn: Stop asserting on |use_gconf| when looking for atk.

|use_gconf| was added back in commit 2f4a32e98 ("linux gn config should
check use_gconf before looking for it with pkg-config") back when the
pkg-config calls to atk and gconf were grouped together within a single
if check.

Since then, commit 95ba4446 ("Move linux pkg_config() calls into
separate BUILD.gn files") has split all the pkg-config checks. As atk
itself does not depend on gconf, we can drop use_gconf from the
assertion in atk's BUILD.gn.

R=brettw@chromium.org,dpranke@chromium.org,mostynb@opera.com
BUG=632297

Review-Url: https://codereview.chromium.org/2241513002
Cr-Commit-Position: refs/heads/master@{#411440}
---
 build/config/linux/atk/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/linux/atk/BUILD.gn b/build/config/linux/atk/BUILD.gn
index 96bf8c0..32eaf88 100644
--- a/build/config/linux/atk/BUILD.gn
+++ b/build/config/linux/atk/BUILD.gn
@@ -6,7 +6,7 @@ import("//build/config/features.gni")
 import("//build/config/linux/pkg_config.gni")
 
 # CrOS doesn't install GTK, gconf or any gnome packages.
-assert(!is_chromeos && use_gconf)
+assert(!is_chromeos)
 
 # These packages should _only_ be expected when building for a target.
 # If these extra checks are not run, gconf is required when building host
-- 
2.7.4

